moveit_ros/planning_interface/move_group_interface/include/moveit/move_group_interface/move_group_interface.h

```diff
@@ -683,6 +685,11 @@ class MoveGroupInterface
       This call is not blocking (does not wait for the execution of the trajectory to complete). */
   MoveItErrorCode asyncMove();
 
+  /** \brief Get the move_group action client used by the \e MoveGroupInterface.
+      The client can be used for querying the execution state of the trajectory and abort trajectory execution
+      during asynchronous execution. */
+  const actionlib::SimpleActionClient<moveit_msgs::MoveGroupAction>& getMoveGroupClient();
```

Comment:
can you actually abort trajectories with a `const` client?
I would have expected this has to be non-const.

Reply:
Yep, you're right, very good catch! `cancelGoal()` was commented out in my test code, so the compiler didn't catch that. Resolved.